db/migrate/20240718150852_activate_admin_style_v3_for_25_p_cent_users.rb

```diff
@@ -0,0 +1,5 @@
+class ActivateAdminStyleV3For25PCentUsers < ActiveRecord::Migration[7.0]
+  def up
+    Flipper.enable_percentage_of_actors(:admin_style_v3, 25)
+  end
+end
```

Comment:
Although it's only intended for server environments, it won't hurt for this to run on dev. I just tested on dev and it remains fully enabled:
![Screenshot 2024-07-22 at 10 24 49 am](https://github.com/user-attachments/assets/e6dc48c8-007b-43ca-a053-6099545a46ac)